modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/configuration/schema/PersistentPageMemoryProfileStorageEngineConfigurationSchema.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.storage.pagememory.configuration.schema;
+
+import org.apache.ignite.configuration.annotation.Config;
+import org.apache.ignite.configuration.annotation.ConfigValue;
+import org.apache.ignite.configuration.annotation.Value;
+import org.apache.ignite.configuration.validation.Immutable;
+import org.apache.ignite.configuration.validation.PowerOfTwo;
+import org.apache.ignite.configuration.validation.Range;
+import org.apache.ignite.internal.pagememory.configuration.schema.PageMemoryCheckpointConfigurationSchema;
+import org.apache.ignite.internal.storage.pagememory.PersistentPageMemoryStorageEngine;
+
+/**
+ * Configuration for {@link PersistentPageMemoryStorageEngine}.
+ */
+@Config
+public class PersistentPageMemoryProfileStorageEngineConfigurationSchema {
```

Comment:
If this is a "Configuration for {@link PersistentPageMemoryStorageEngine}", then why isn't this class called `PersistentPageMemoryStorageEngineConfigurationSchema`?

Reply:
Will be renamed in the next PR, when the legacy configuration will be removed https://issues.apache.org/jira/browse/IGNITE-21385